atmosphere/um_env.yaml

```diff
@@ -0,0 +1,46 @@
+AINITIAL: ''
+ARCLBIOG: INPUT/biogenic_351sm.N96L38
+ASTART: INPUT/pre-industrial.astart
+AUSCOM_CPL: 'true'
+BIOMASS: INPUT/Bio_1850_ESM1.anc
+CHEMOXID: INPUT/sulpc_oxidants_N96_L38
+DATAW: '{work_path}'
+DMSCONC: INPUT/DMS_conc.N96
+FASTRUN: 'true'
+IDEALISE: ''
+NDEPFIL: INPUT/Ndep_1850_ESM1.anc
+OCFFEMIS: INPUT/OCFF_1850_ESM1.anc
+OZONE: INPUT/ozone_1850_ESM1.anc
+PAREXE: parexe
+PRINT_STATUS: PrStatus_Diag
+RPSEED: ''
+RUNID: PI-01
+SOOTEMIS: INPUT/BC_hi_1850_ESM1.anc
+STASETS_DIR: INPUT/stasets
+SULPEMIS: INPUT/scycl_1850_ESM1_v4.anc
+TYPE: NRUN
+UM_ATM_NPROCX: '16'
+UM_ATM_NPROCY: '12'
+UM_NAM_MAX_SECONDS: '300'
+UM_NPES: '192'
+UM_SECTOR_SIZE: '2048'
+UM_STDOUT_FILE: atm.fort6.pe
+UNIT01: ''
+UNIT02: prefix
+UNIT04: STASHC
+UNIT05: namelists
+UNIT07: PI-01.out2
+UNIT08: /dev/null
+UNIT09: CONTCNTL
+UNIT10: xhist
+UNIT11: ihist
+UNIT12: thist
+UNIT14: errflag
+UNIT15: ''
+UNIT22: INPUT/STASHmaster
+UNIT57: INPUT/spec3a_sw_hadgem1_6on
+UNIT58: ''
+UNIT80: INPUT/spec3a_lw_hadgem1_6on
+VEGINIT: INPUT/cable_vegfunc_N96.anc
+VERT_LEV: INPUT/vertlevs_G3
+VN: '7.3'
```

Comment:
Just realised the yaml conversion script I put together inadvertently rearranges this file to alphabetical order... I'll try to fix this in the script.  Would you be happy to use the below order which duplicates the original file and also adds back in the original comments (which the script also strips)?


```suggestion
AINITIAL: ''
ASTART: INPUT/pre-industrial.astart
AUSCOM_CPL: 'true'
DATAW: '{work_path}'
FASTRUN: 'true'
IDEALISE: ''
PAREXE: parexe
PRINT_STATUS: PrStatus_Diag
RPSEED: ''
RUNID: PI-01
TYPE: NRUN
UM_ATM_NPROCX: '16'
UM_ATM_NPROCY: '12'
UM_NAM_MAX_SECONDS: '300'
UM_NPES: '192'
UM_SECTOR_SIZE: '2048'
UM_STDOUT_FILE: atm.fort6.pe
VN: '7.3'

# Config input paths
UNIT01: ''
UNIT02: prefix
UNIT04: STASHC
UNIT05: namelists
UNIT07: PI-01.out2
UNIT08: /dev/null
UNIT09: CONTCNTL
UNIT10: xhist
UNIT11: ihist
UNIT12: thist
UNIT14: errflag
UNIT15: ''
UNIT58: ''
UNIT22: INPUT/STASHmaster
UNIT57: INPUT/spec3a_sw_hadgem1_6on
UNIT80: INPUT/spec3a_lw_hadgem1_6on
STASETS_DIR: INPUT/stasets
VERT_LEV: INPUT/vertlevs_G3

# Ancillary files
ARCLBIOG: INPUT/biogenic_351sm.N96L38
BIOMASS: INPUT/Bio_1850_ESM1.anc
CHEMOXID: INPUT/sulpc_oxidants_N96_L38
DMSCONC: INPUT/DMS_conc.N96
NDEPFIL: INPUT/Ndep_1850_ESM1.anc
OCFFEMIS: INPUT/OCFF_1850_ESM1.anc
OZONE: INPUT/ozone_1850_ESM1.anc
SOOTEMIS: INPUT/BC_hi_1850_ESM1.anc
SULPEMIS: INPUT/scycl_1850_ESM1_v4.anc
VEGINIT: INPUT/cable_vegfunc_N96.anc
```

Reply:
Sure.

Reply:
Done.

Reply:
Thanks for doing that and apologies about that not being in the script originally!